pkg/util/child.go

```diff
@@ -0,0 +1,30 @@
+// +build !linux
+
+/*
+Derived from:
+
+https://github.com/kubernetes/kubernetes
+
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+)
+
+// childSignals are the allowed signals that can be sent to children in Windows to terminate
```

Comment:
yes I saw it, but i wasn't going to hold up this PR while that PR shakes
out.  If/when that PR is resolved, we can come back and update this.


On Tue, Apr 5, 2016 at 5:16 PM, Rodolfo Carvalho <notifications@github.com>
wrote:

> In pkg/util/child.go
> <https://github.com/openshift/source-to-image/pull/452#discussion_r58615592>
> :
>
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package util
> > +
> > +import (
> > +	"os"
> > +)
> > +
> > +// childSignals are the allowed signals that can be sent to children in Windows to terminate
>
> @bparees <https://github.com/bparees> what do you think about this? Have
> you seen kubernetes/kubernetes#23865
> <https://github.com/kubernetes/kubernetes/pull/23865>?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/source-to-image/pull/452/files/50b50bda3e4f3867f66332cf9747b7c4973660f4#r58615592>
>



-- 
Ben Parees | OpenShift